pkg/tsdb/mysql/sqleng/sql_engine.go

```diff
@@ -300,14 +304,14 @@ func (e *DataSourceHandler) executeQuery(query backend.DataQuery, wg *sync.WaitG
 	}
 
 	if err := convertSQLTimeColumnsToEpochMS(frame, qm); err != nil {
-		errAppendDebug("converting time columns failed", err, interpolatedQuery)
+		errAppendDebug("converting time columns failed", err, interpolatedQuery, backend.ErrorSourcePlugin)
 		return
 	}
 
 	if qm.Format == dataQueryFormatSeries {
 		// time series has to have time column
 		if qm.timeIndex == -1 {
-			errAppendDebug("db has no time column", errors.New("no time column found"), interpolatedQuery)
+			errAppendDebug("db has no time column", errors.New("no time column found"), interpolatedQuery, backend.ErrorSourceDownstream)
```

Comment:
I'm not sure if it is a downstream error. This is something that the plugin throws because of no time column found.